fplus-lib/src/external_services/github.rs

```diff
@@ -156,12 +151,25 @@ impl GithubWrapper {
             }))
             .build()
             .expect("Could not create Octocrab instance");
-        let iod: InstallationId = installation_id.into();
-        let installation = octocrab.installation(iod);
-        Self {
-            owner,
-            repo,
-            inner: Arc::new(installation),
+        if installation_id.is_some() {
+            let pared_istallation_id: u64 = installation_id
```

Comment:
```suggestion
            let installation_id: u64 = installation_id
```
Rationale:
1. Original has 2 typos
2. We're not changing the meaning of the variable and we don't need to keep the old one, so we can just reuse the name.